drawBot/context/tools/imageObject.py

```diff
@@ -119,14 +130,14 @@ def unlockFocus(self):
         pageCount = data.pageCount()
         page = data.pageAtIndex_(pageCount - 1)
         # create an image
-        im = AppKit.NSImage.alloc().initWithData_(page.dataRepresentation())
+        im = AppKit.NSImage.alloc().initWithData_(page.dataRepresentation()) # type: ignore
```

Comment:
I don't remember why I added the comment, but all AppKit is dynamic, so static analysis does not work.

Reply:
That doesn't answer my question, though.

Reply:
Apparently these `# type: ignore` after `AppKit` module calls are not necessary since we are ignoring it directly at the import statement. Fixed with c41c63b5f825d4f38bcd4939d1ea815e6ca1555d